lib/routes/register/index.js

```diff
@@ -170,6 +170,15 @@ async function register(server, options, done) {
           throw new Error()
         }
       } catch {
+        await server.pg.write.query(
+          metricsInsert({
+            event: 'TOKEN_RENEWAL_FAILED',
+            os: '',
+            version: '',
+            timeZone: options.timeZone
+          })
+        )
+
         throw new Unauthorized('Refresh token is missing')
```

Comment:
This should capture the TOKEN_RENEWAL_FAILED also, any criteria that results in us not issuing a token should be captured in the metric